pkg/controller/node/nodecontroller.go

```diff
@@ -340,6 +351,24 @@ func (nc *NodeController) Run(period time.Duration) {
 	}, nodeEvictionPeriod, wait.NeverStop)
 
 	go wait.Until(nc.cleanupOrphanedPods, 30*time.Second, wait.NeverStop)
+
+	for i := 0; i < cidrUpdateWorkers; i++ {
+		go func(stopChan <-chan struct{}) {
+			for {
+				select {
+				case workItem, ok := <-nc.nodeCIDRUpdateChannel:
+					if !ok {
+						glog.Warning("NodeCIDRUpdateChannel read returned false.")
+						break
+					}
+					nc.updateCIDRAllocation(workItem)
+				case <-stopChan:
+					glog.Warning("StopChannel is closed.")
```

Comment:
This shouldn't be warning - if someone is explicitly closing stopChan, we shouldn't log it as warning.